spec/paper_trail/model_spec.rb

```diff
@@ -264,11 +264,18 @@
           expect(widget.versions.last.item).to be_nil
         end
 
-        it "not have changes" do
-          widget = Widget.create(name: "Henry")
-          widget.update_attributes(name: "Harry")
-          widget.destroy
-          expect(widget.versions.last.changeset).to eq({})
+        it "has changes" do
+          book = Book.create! title: "A"
+          changes = YAML.load book.versions.last.attributes["object_changes"]
+          expect(changes).to eq "id" => [nil, book.id], "title" => [nil, "A"]
```

Comment:
The predominant code style for this repo uses an additional set of parentheses for the `eq` matcher, and sometimes for the `to` as well.

```ruby
expect(changes).to eq("id" => [nil, book.id], "title" => [nil, "A"])
```

In my experience, using parentheses for neither is hard for many people to follow.